Refresh.GameServer/Refresh.GameServer.csproj

```diff
@@ -19,6 +20,11 @@
     
     <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
         <PublishSingleFile>true</PublishSingleFile>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    </PropertyGroup>
+    
+    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
All this should be at the leading `PropertyGroup` at the top of the file.

Reply:
rider moment, will fix